### PR TITLE
Closes #942 Update ASLDRO

### LINKED ISSUE #942: Update all DRO unit tests (new v2.3.0 structure)
### Description

Update all unit tests which use the ASLDRO. They should use the new BIDS file structure and the new filenames.

- [Related commit a](https://github.com/ExploreASL/Testing/commit/e87268ff76ce74ce5cdf438e3c27f702268fc795)
- [Related commit b](https://github.com/ExploreASL/Testing/commit/e6cc8ce1778eef4ffaf9e0c67da98d6e8ab9cc61)

### Release notes

Update unit tests for ASL-DRO.

### PR DESCRIPTION
**Pull request by MichaelStritt**
### Linked issue

Check out #942

### How to test

```matlab
[UnitTests,UnitTestsTable] = xASL_test_UnitTesting;
```

### CODE REVIEW

**Comment by MichaelStritt** [comment]
@jan-petr: `xASL_ut_master_xASL_ExploreASL_Master.m` still needs some work and I stumbled upon the bug that Henk mentioned a while ago. Determining the subject/session/run structure without a sourceStructure.json does not seem to work right now, which is why the sub-test 5 of `xASL_ut_master_xASL_ExploreASL_Master.m` fails. Does this need to be fixed or are we going to make everything purely DCM->BIDS in the future?

File: Testing/UnitTests/xASL_ut_master_xASL_ExploreASL_Master.m [hunk 1]
```diff
@@ -264,7 +264,7 @@
 testTime = tic;
 
 % Copy test patient
-testPatientSource = fullfile(TestRepository,'UnitTesting','synthetic_dcm','test_patient_2_2_0'); 
+testPatientSource = fullfile(TestRepository,'UnitTesting','synthetic_dcm','test_patient_2_2_0');
```
Comment (jan-petr) [inline Testing/UnitTests/xASL_ut_master_xASL_ExploreASL_Master.m:267]: 2_2_0 is on purpose here? Shouldn't we rename the input files?

**Review by jan-petr — CHANGES_REQUESTED** [review]
Looks good. Two minor things:

- Please update the `str` to `string` at line Functions/xASL_bids_Config.m:15:% 
(my fault for not fixing it previously)
- one comment in the code

File: Testing/UnitTests/xASL_ut_master_xASL_ExploreASL_Master.m (hunk 1 above)
Comment (MichaelStritt) [reply Testing/UnitTests/xASL_ut_master_xASL_ExploreASL_Master.m:267]: I once created some synthetic dicoms of the asldro 2.2.0. I don't have that from 2.3.0 yet.

Comment (jan-petr) [reply Testing/UnitTests/xASL_ut_master_xASL_ExploreASL_Master.m:267]: I see. Good. Can you then create a new issue for doing that in the future?

**Review by jan-petr — APPROVED** [review]
OK

File: Testing/UnitTests/xASL_ut_master_xASL_ExploreASL_Master.m (hunk 1 above)
Comment (MichaelStritt) [reply Testing/UnitTests/xASL_ut_master_xASL_ExploreASL_Master.m:267]: Yep

**Pull request merged**